foursight_core/app_utils.py

```diff
@@ -131,6 +155,78 @@ def init_response(self, environ):
             response.status_code = 400
         return connection, response
 
+    def is_running_locally(self, request_dict):
+        return request_dict.get('context', {}).get('identity', {}).get('sourceIp', '') == "127.0.0.1"
+
+    def get_logged_in_user_info_from_jwt_token_cookie(self, request_dict):
+        email_address = ""
+        email_verified = ""
+        first_name = ""
+        last_name = ""
+        issuer = ""
+        subject = ""
+        audience = ""
+        issued_time = ""
+        expiration_time = ""
+        jwt_decoded = ""
+        try:
+            jwt_token = self.get_jwt(request_dict)
+            if jwt_token:
+                options = {"verify_signature": False, "verify_aud": False}
```

Comment:
Is there a reason we don't want to verify the signature?

Reply:
Oh, good call, I should be using the other code that actually passess the Auth0 client ID and secret. Will refactor.